libraries/config/include/config/state_api_config.hpp

```diff
@@ -42,9 +42,12 @@ Json::Value enc_json(DPOSConfig const& obj);
 void dec_json(Json::Value const& json, DPOSConfig& obj);
 
 struct ExecutionOptions {
-  bool disable_nonce_check = 0;
-  bool disable_gas_fee = 0;
-  bool enable_nonce_skipping = 0;
+  bool disable_nonce_check = false;
+  bool disable_gas_fee = false;
+  bool enable_nonce_skipping = false;
+  // Do not process reward pool (newly minted tokens & tx fees) according to the reward statistics,
+  // give whole reward pool to the pbft block proposer
+  bool disable_stats_rewards = false;
```

Comment:
hmm maybe irrelevant comment but maybe this param should be part of same config as `disable_block_rewards`